dali/server/daldap.cpp

```diff
@@ -186,13 +184,22 @@ class CDaliLdapConnection: implements IDaliLdapConnection, public CInterface
                     return SecAccess_None;//deny
                 }
 
-                authenticated = true;//Digital signature verified
+                //Mark user as authenticated. The call below to authenticateUser
+                //will add this user to the LDAP cache
+                user->setAuthenticateStatus(AS_AUTHENTICATED);
+                if (isEmptyString(user->credentials().getSignature()))
+                {
+                    //Set user digital signature
+                    StringBuffer b64Signature;
+                    pDSM->digiSign(user->getName(), b64Signature);
+                    user->credentials().setSignature(b64Signature);
+                }
             }
             else
                 ERRLOG("LDAP: getPermissions(%s) scope=%s user=%s digital signature support not available",key?key:"NULL",obj?obj:"NULL",username.str());
         }
 
-        if (!authenticated && !ldapsecurity->authenticateUser(*user, NULL))
+        if (!ldapsecurity->authenticateUser(*user, NULL))
```

Comment:
Why does it need to call authenticateUser?  From a quick scan of the code in authorizeFileScope it should still go over to ldap to resolve to rights (even though it will be inefficient).  Please add more details of why it fails to the jira - there isn't really enough information there to check the PR fixes the problem